notebooks/animate-anyone/animate-anyone.ipynb

```diff
@@ -0,0 +1,1652 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    os.makedirs(local_dir, exist_ok=True)
</pre><p>why do you mix os.path and Path objects, all used in this cell path-oriented commands can be performed with Path object without using additional libs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1846/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>